diff --git a/node_modules/expo-keep-awake/src/index.ts b/node_modules/expo-keep-awake/src/index.ts
index b7a856e..44c5bcb 100644
--- a/node_modules/expo-keep-awake/src/index.ts
+++ b/node_modules/expo-keep-awake/src/index.ts
@@ -34,14 +34,10 @@ export function useKeepAwake(tag?: string, options?: KeepAwakeOptions): void {
       if (isMounted && ExpoKeepAwake.addListenerForTag && options?.listener) {
         addListener(tagOrDefault, options.listener);
       }
-    });
+    }).catch(() => {});
     return () => {
       isMounted = false;
-      if (options?.suppressDeactivateWarnings) {
-        deactivateKeepAwake(tagOrDefault).catch(() => {});
-      } else {
-        deactivateKeepAwake(tagOrDefault);
-      }
+      deactivateKeepAwake(tagOrDefault).catch(() => {});
     };
   }, [tagOrDefault]);
 }
